module4/projects/capstone/evaluation.md

```diff
@@ -0,0 +1,185 @@
+</section>
+
+<section class="dropdown">
+### Work Ethic and Attitude
+- **Exceeds Expectations:**
+  * Regularly brings questions and ideas to check-ins
+  * Responds with an appropriate urgency to messages in Slack during work hours
+  * Demonstrates a positive attitude and willingness to learn
+  * Takes initiative to research and propose solutions to problems
+  * Completes self assessment with thoughtful reflection and clear action items
+- **Meets Expectations:**
+  * Attends every live or async daily check-in, on time, and notifies the instructor if they are unable to attend
+  * Actively participates in daily stand-ups, providing clear updates on progress and blockers
+  * Responds to Slack messages within a reasonable amount of time during work time
+  * Shows enthusiasm for the project and a desire to improve
+  * Maintains a positive and respectful attitude in all team interactions
+  * Completes self assessment with thoughtful reflection
+- **Approaching Expectations:**
+  * Misses or is late to up to two live or async daily check-ins without notifying the instructor
+  * Does not always provide clear updates on progress and blockers during daily check-ins
+  * Occasionally responds to Slack messages slowly such that other teammates are blocked
+  * Slow to ask questions
+  * Actively participates in the project but does just enough to get by
+  * Completes self assessment but reflection is lacking
+- **Below Expectations:**
+  * Misses or is late to three or more live or async daily check-ins without notifying the instructor
+  * Daily updates are unclear, vague, or lacking detail at least half the time
+  * Asks questions infrequently, if at all
+  * Sporadic or infrequent communication in Slack
+  * Does not participate in the project or does not contribute to the project in a meaningful way
+  * Does not fill out the self assessment
+</section>
+
+<section class="dropdown">
+### Collaborative Workflow
+- **Exceeds Expectations:**
+  * Consistently updates project board and includes progress and blockers
+  * Provides constructive and actionable feedback on 4 or more pull requests
+  * Uses co-authored commits on paired pull requests
+  * Solicits project ideas and feedback from quieter teammates
+- **Meets Expectations:**
+  * Moves tasks along on the project board in real time (or close to it)
+  * Communicates effectively with team members and project manager through appropriate channels
+  * Collaborates well on pair programming tasks, showing both the ability to lead and follow
+  * Provides constructive and actionable feedback on at least 3 pull requests
+  * Seeks clarification when requirements or tasks are unclear
+  * Demonstrates active listening skills during team discussions and meetings
+  * Shows willingness to help team members when they encounter difficulties
```

Comment:
Could we add something here like "transparently communicates blockers and seeks help when appropriate" as a compliment to this bullet point? I think both could be valuable to encourage. I see it also builds on the approaching expectation bullet around asking for help too?